src/app/app-routing.module.ts

```diff
@@ -5,6 +5,7 @@ import {RoomComponent} from './components/room/room.component';
 import {CreateTaskComponent} from './components/create-task/create-task.component';
 import {RoomComponent as Host} from './components/product-owner/room/room.component';
 import {ParticipantRoomComponent} from './components/participant/participant-room/participant-room.component';
+import {SummaryComponent} from "./components/summary/summary.component";
```

Comment:
Is it possible to set the `components` folder as a resource root? This way you could do just `components/summary/summary.component` - might come in handy when the file that needs a component is really far away in structure. Just an idea, if not - ignore this comment

Reply:
This isn't a good practice in Angular I think. The components folder is created by me, and there are others folders like services, classes - I'm importing from them as well.

Reply:
Sure, I don't know the full structure of your project. I kind of assumed (wrongly) here that it is probably as we do in React - you create a shared folder called `components` which contains reusable components to be used in the whole system, like buttons etc ;)

Reply:
👍